docker/init_trino.bash

```diff
@@ -7,6 +7,5 @@ cd ..
 set -exo pipefail
 
 docker-compose -f docker-compose-trino.yml build
-docker-compose -f docker/util.yml build
 docker-compose -f docker-compose-trino.yml up -d
-docker-compose -f docker/util.yml run --rm util wait_for_up trino 8080
+while ! docker-compose -f docker-compose-trino.yml logs trino 2>&1 | tail -n 1 | grep "SERVER STARTED"; do sleep 2; done
```

Comment:
Why did you opt to not have timeout option anymore?

Reply:
added timeout